eahub/templates/eahub/group.html

```diff
@@ -147,6 +147,14 @@ <h1 class="word-break">{{ group.name }}</h1>
   </div>
   {% endif %}
 
+
+  <div class="text-center margin-top report-btn">
+    <div class="btn-group btn-group-sm text-center" role="group" id="report_profile_abuse_toggle">
+      <a href="{% url 'report_abuse_group' group.slug  %}"><button type="button" class="btn btn-default"><i class="fa fa-flag"></i> Report abuse</a>
+    </div>
+  </div>
+  
+
 </div>
-<script src="/static/scripts/group.js"></script>
+<script src="/static/scripts/report_buttons.js"></script>
```

Comment:
Use the `{% static %}` tag to link to static assets. It makes the site faster, because Whitenoise makes them cacheable.